src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -282,6 +283,11 @@ CPU::CPU() {
         OPENVINO_THROW("CPU affinity check failed. No CPU is eligible to run inference.");
     };
 
+    if (_proc_type_table.size() > 1) {
+        int cur_processor_id = sched_getcpu();
+        update_table_for_proc(cur_processor_id, _proc_type_table, _cpu_mapping_table);
```

Comment:
1. Too generic name? It is mostly like: sort_table_by_cpu_id
2. We have to add comment why this sort is needed at all.

Reply:
Updated function name and added function description.